aiida/backends/tests/cmdline/commands/test_import.py

```diff
@@ -191,13 +185,8 @@ def test_import_old_url_archives(self):
         self.assertIn("Success: imported archive {}".format(options[0]), result.output, msg=result.exception)
 
     def test_import_url_and_local_archives(self):
-        """Test import of both a remote and local archive
-        TODO: Update 'url' to point at correct commit and file.
-        Now it is pointing to yakutovicha's commit, but when PR #2478 has been merged in aiidateam:develop,
-        url should be updated to point to the, essentially same, commit, but in aiidateam.
-        Furthermore, the filename should be change from '_no_UPF.aiida' to '_simple.aiida'.
-        """
-        url = "https://raw.githubusercontent.com/yakutovicha/aiida_core/f5fff1846a62051b898f13db67f5eef18892d5f4/"
+        """Test import of both a remote and local archive"""
+        url = "https://raw.githubusercontent.com/aiidateam/aiida_core/0599dabf0887bee172a04f308307e99e3c3f3ff2/"
```

Comment:
as we are repeating ourselves here, perhaps better to make this a variable inside this module.
since it's only two occurences it's OK for the moment but in the future there might be more, so one can already put the variable

Reply:
done